pycbc/workflow/jobsetup.py

```diff
@@ -908,10 +906,7 @@ class PycbcSplitInspinjExecutable(Executable):
     The class responsible for running the pycbc_split_inspinj executable
     """
     current_retention_level = Executable.INTERMEDIATE_PRODUCT
-    def __init__(self, cp, exe_name, num_splits, universe=None, ifo=None,
-                 out_dir=None):
-        super().__init__(cp, exe_name, universe, ifo, out_dir, tags=[])
-        self.num_splits = int(num_splits)
```

Comment:
I think this will break this one. num_splits is needed

Reply:
You are right.